mocca-client/src/main/java/com/paypal/mocca/client/MoccaValidator.java

```diff
@@ -0,0 +1,7 @@
+package com.paypal.mocca.client;
+
+public interface MoccaValidator {
```

Comment:
I don't see this being used anywhere. Is this residual code from when you were playing with the idea of an SPI?

Reply:
that should have been deleted!